guides/source/api_app.md

```diff
@@ -374,8 +372,7 @@ controller modules by default:
 - `ActionController::Rescue`: Support for `rescue_from`.
 - `ActionController::Instrumentation`: Support for the instrumentation
   hooks defined by Action Controller (see [the instrumentation
-  guide](active_support_instrumentation.html#action-controller) for
-more information regarding this).
+  guide](active_support_instrumentation.html#action-controller) for more information regarding this).
```

Comment:
I don't see a change here except the lines merged into one. Was that the intent? No need, imo.